src/test/java/com/microsoft/sqlserver/testframework/AbstractTest.java

```diff
@@ -99,6 +111,90 @@ public static void setup() throws Exception {
         }
     }
 
+    /**
+     * Covers only connection properties required for testing. Does not cover all connection properties - add more
+     * properties if needed.
+     * 
+     * @param ds
+     *        DataSource to be configured
+     * @return ISQLServerDataSource
+     */
+    private static ISQLServerDataSource updateDataSource(ISQLServerDataSource ds) {
```

Comment:
I would suggest to use `enum SQLServerDriverStringProperty` and `java.util.Properties.getProperty()`, similar to what we already do in the driver to parse the connection string.

Reply:
They're private/protected..

Reply:
We can keep copies of the `enum`'s in the test framework too, would be cleaner than listing them in a `switch`, which would also let us use `java.util.Properties` to do the parsing.

Reply:
I tried with Enums, but we can't use Enums in switch cases, hence I'm going to Constants.. also cleaning up staggered constants across all tests.. do you want this change in this PR?

Reply:
Why do you need the switch case? Just like in the driver, if a property is not specified, `java.util.Properties.getProperty()` will return `null`.

Reply:
I would prefer a separate PR for those changes.

Reply:
Can you change the method signature to `updateDataSource(String connectionString, ISQLServerDataSource ds)`? This will be useful in case if we have another connection string specified, NTLM for example.

Reply:
Sure. I'll push that update.